zowe/zowe-cli
# Use Fewer Deprecated Functions in Master

**What It Does**

Resolves #2191
Reduces the use of deprecated functions in preparation to support Node 22

**How to Test**

Install, build, run commands to verify new build does not trigger deprecation warnings on Node 22

**Review Checklist**
I certify that I have:


**Additional Comments**
